ethereum/swap_creator_test.go

```diff
@@ -31,6 +31,7 @@ var (
 	defaultTimeoutDuration = big.NewInt(60) // 60 seconds
 	ethAssetAddress        = ethcommon.Address(types.EthAssetETH)
 	defaultSwapValue       = big.NewInt(100)
+	fakeSwapKey            = [32]byte{1} // dummy non-zero value for claim/refund key
```

Comment:
I like the name in the comment better than the one on the variable.  "dummy" vs "fake".